2023/did-wg.html

```diff
@@ -337,13 +315,18 @@ <h2>Success Criteria</h2>
 more implementations interoperating with each other. In order to advance to
 Proposed Recommendation, each normative specification must have an open
 test suite of every feature defined in the specification.</p>
+
+    <p>In order for <a href="#deliverable-did-resolution">DID Resolution</a> to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, it is expected to define a dummy DID method meant only for demonstrating interoperability across independent implementations. While required for all complying implementations, this method will be explicitly described as unfit for use in production.
+    It is also expected that the Working Group can provide evidence of at least two independent open specifications of DID methods being developed outside of the Working Group, and relying on <a href="#deliverable-did-core">DID Core</a> and <a href="#deliverable-did-resolution">DID Resolution</a>.
```

Comment:
```suggestion
    <p>In order for <a href="#deliverable-did-resolution">DID Resolution</a> to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, it is expected that each of the independant implementations mentioned above support at least two DID methods with an open specification (i.e. a specification that is accessible to all for implementation and deployment). It is also expected that each pair of the independant implementations mentioned above support at least one common DID method with an open specification.</p>
```
@OR13  @rxgrant would this change work for you?
It gets rid of the dummy DID method (which, in my understanding) was not a strong requirement from anybody at TPAC, just a suggestion),
and it changes the focus on DID method to consider them as part of the implementations.
It shifts the "choice" of the DID methods further away from the WG towards the implementers, hopefully removing any risk of a perceived endorsement by the WG.

I'm not entirely sure that this change would fly with everyone else involved in the discussion, so I will commit it only if you tell me that it is acceptable for you.

Reply:
@pchampin Why would you default to a version that is definitely unacceptable to me, if others object?  What is your technical reason for doing so?

What might be the technical reason for objecting coming from others?  Are they part of this conversation?

Reply:
I also like this change. I think the idea of defining a "dummy DID method" has good intentions, but I'm worried it will distract the work of the group. At TPAC, a strong desire for "practical interoperability" was expressed, and I'm convinced we can easily demonstrate such interoperability without actually defining a DID method in the WG.

Reply:
I agree with @peacekeeper's comments and sentiments.

Reply:
> @pchampin Why would you default to a version that is definitely unacceptable to me, if others object? What is your technical reason for doing so?

@rxgrant I never said that I would default to the current version if other people objected to the change above. All I wrote was that I would not make the change if it didn't satisfy you and @OR13  more than the current version. I see no reason (technical or not) to make a change that risks creating more objections if it does not help solving the existing objections...

But in fact, as you confirm that it is acceptable to you, I'll try my luck with this change.